src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -0,0 +1,142 @@
+
+Adding `rel=preconnect` to a `<link>` informs the browser that your page intends to establish a connection to another domain, and that you'd like the process to start as soon as possible. Resources will load more quickly because the setup process has already been completed by the time the browser requests them. 
+
+Resource hints are named so because they are not mandatory instructions. They provide the information about what you'd like to happen, but it's ultimately up to the browser to decide whether to execute them. Setting up and keeping a connection open is a lot of work, so the browser might choose to ignore resource hints or execute them partially depending on the situation.
+
+Informing the browser of your intention is as simple as adding a `<link>` tag to your page:
+
+```html
+<link rel="preconnect" href="https://example.com">
+```
+
+![A diagram showing how the download doesn't start for a while after the connection is established.](preconnect.png)
+
+You can speed up the load time by 100–500 ms by establishing early connections to important third-party origins. These numbers might seem small, but they make a difference in how [users perceive web page performance](https://developers.google.com/web/fundamentals/performance/rail#ux).
+
+{% Aside %}
+chrome.com [improved Time To Interactive](https://twitter.com/addyosmani/status/1090874825286000640) by almost 1 s by pre-connecting to important origins. An e-commerce website [improved product image load time by 500 ms](https://andydavies.me/blog/2019/03/22/improving-perceived-performance-with-a-link-rel-equals-preconnect-http-header/) by pre-connecting to the image CDN.
+{% endAside %}
+
+## Use-cases
+
+### Knowing *where from*, but not *what* you're fetching
+
+Due to versioned dependencies, you sometimes end up in a situation where you know you'll be requesting a resource from a particular CDN, but not the exact path for it. 
+
+<figure class="w-figure w-figure--center">
+<img src="versioned-url.png" style="max-width:450px" alt="A url of a script with the version name.">
+<figcaption>An example of a versioned URL</figcaption>
+</figure>
+
+The other common case is loading images from an [image CDN](/image-cdns), where the exact path for an image depends on media queries or runtime feature checks on the user's browser.
+
+<figure class="w-figure w-figure--center">
+<img src="image-cdn-url.png" alt="An image CDN URL with the parameters size=300x400 and quality=auto.">
+<figcaption>An example of an image CDN URL</figcaption>
+</figure>
+
+In these situations, if the resource you'll be fetching is important, you want to save as much time as possible by pre-connecting to the server. The browser won't download the file before it needs it (that is, once the request is made from your page somehow), but at least it can handle the connection aspects ahead of time, saving the user from waiting for several round trips.
```

Comment:
```suggestion
In these situations, if the resource you'll be fetching is important, you want to save as much time as possible by pre-connecting to the server. The browser won't download the file until your page requests it, but at least it can handle the connection aspects ahead of time, saving the user from waiting for several round trips.
```
I had a little trouble parsing the aside on the first read, so tried to simplify.